docs/courses/csintro3/orientation/math.md

```diff
@@ -1,101 +1,171 @@
 # Math
 
-table with math blocks -> equivalent
+When transitioning into JavaScript, we see that doing calculations and writing math formulas becomes a lot easier.
 
-| blocks | JavaScript |
-| -- | -- |
-| x  |  * |
-| +  |  + | 
-| %  |  / | 
+## Math Operators
 
-etc
+When writing code with blocks, we saw that we needed to have a block for each math operator (e.g. +, -, &times;, &divide;). This can be pretty annoying and make our code formulas more complicated than they need to be. We'll see that in JavaScript, formulas are a lot more condensed.
+
+Here is how we translate each operator from blocks to JavaScript
+
+|Operation 					|Block					|Javascript			|
+|:---						|:---:					|:---				|
+|Addition (+)				|`[let x = (6 + 2)]`	|`let x = (6 + 2);`	|
+|Subtraction (-)			|`[let x = (6 - 2)]`	|`let x = (6 - 2);`	|
+|Multiplication (&times;)	|`[let x = (6 * 2)]`	|`let x = (6 * 2);`	|
+|Division (&divide;)		|`[let x = (6 / 2)]`	|`let x = (6 / 2);`	|
 
 ### Complex Expressions
 
-```typescript
-let num: number = 5 + 3 * 2;
-console.log("" + num);
+As mentioned above, when using blocks, we needed to use a separate block for each Math operator. In JavaScript, it is a lot easier to string these operators together.
+
+For example, what was 
+
+```blocks
+let num = 0
+num = 1 + (2 + 3)
 ```
 
+is now
+
 ```typescript
-let num: number = 15 / 3 * 8;
+let num: number = 1 + 2 + 3
 console.log("" + num);
 ```
 
-## Assignment Operators
-
-### Show increment / change by transition (blocks -> JavaScript)
+Some other examples are:
 
 ```typescript
-let num: number = 0;
-num = num + 5;
-num = num + 3;
+let num: number = 5 + 3 * 2;
+console.log("" + num);
 ```
 
 ```typescript
-let num: number = 0;
-num += 5;
-num += 3;
+let num: number = 24 / 3 * 4;
+console.log("" + num);
 ```
 
-### Special case: add or subtract 1 (increment / decrement operators)
+## Order of Operations
+
+Think about how these might be ambiguous. What value do these expressions evaluate to? It depends based on which operation you do first. As it turns out, JavaScipt uses a **PEMDAS** structure to determine which operations should be done in what order.
+
+|Order	|Operation 		|
+|:---:	|:---			|
+|1st	|Parenthesis	|
+|2nd	|Exponents		|
+|3rd	|Multiplication	|
+|		|Division		|
+|4th	|Addition	 	|
+|		|Subtraction 	|
+
+Operations that are of the same order (e.g. multiplication and division) will be completed left-to-right. That is ```24 / 3 * 4``` evaluates to 32 because 24 is divided by 3 before the multiplication happens.
+
+While parenthesis are not an operator, they serve as a way to group operations together. Operations done within parenthesis will be completed first before moving onto those outside.
```

Comment:
parenthesis is singular (a matching pair); should be "While parentheses are not an operator".

Maybe rephrase slightly, eg something likely

Any operations that occur within parentheses apply first in the order of operations. This can be used to control the order in which an expression is evaluated.